installer/bundle/bundle_skel.sh

```diff
@@ -302,15 +302,18 @@ ulinux_detect_openssl_version()
     OPENSSL_SYSTEM_VERSION_10X=`echo $OPENSSL_SYSTEM_VERSION_FULL | grep -Eq '^1.0.'; echo $?`
     OPENSSL_SYSTEM_VERSION_100_ONLY=`echo $OPENSSL_SYSTEM_VERSION_FULL | grep -Eq '^1.0.0'; echo $?`
     OPENSSL_SYSTEM_VERSION_11X=`echo $OPENSSL_SYSTEM_VERSION_FULL | grep -Eq '^1.1.'; echo $?`
+    OPENSSL_SYSTEM_VERSION_30X=`echo $OPENSSL_SYSTEM_VERSION_FULL | grep -Eq '^3.0.'; echo $?`
```

Comment:
basically we package 3 versions each of deb/rpm for 1.0.0, 1.1.0 and 3.0 and install based on what is available in cx vm? Is this correct? How much has package size increased to now?

Reply:
yes, total bundle size is now about 235MB. Previously it was about 155MB , increase is due to 2 ruby and adding extra packages for 300 omi,scx, omsagent, dsc.